gridstatus/tests/source_specific/test_ercot_api.py

```diff
@@ -1436,3 +1436,40 @@ def _endpoints_map_check(self, endpoint_dict: dict) -> list[str]:
                 elif not callable(parser_method):
                     issues.append(f"{param} has an invalid parser_method")
         return issues
+
+    @pytest.mark.parametrize(
+        "date, end",
+        [
+            ("2024-02-15 00:00:00", "2024-02-15 01:00:00"),
+            ("2024-03-10 00:00:00", "2024-03-10 04:00:00"),
+            ("2024-11-03 00:00:00", "2024-11-03 02:00:00"),
+        ],
+    )
+    def test_get_indicative_lmp_by_settlement_point(self, date, end):
+        with api_vcr.use_cassette(
+            f"test_get_indicative_lmp_historical_{date}_{end}.yaml",
+        ):
+            df = self.iso.get_indicative_lmp_by_settlement_point(date, end)
+
+            assert df.columns.tolist() == [
+                "Interval Start",
+                "Interval End",
+                "RTDTimestamp",
```

Comment:
need to add space here and in `test_ercot.py`

<img width="838" alt="image" src="https://github.com/user-attachments/assets/9438bf0e-2f2e-418b-87d1-cffd99976287" />